src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java

```diff
@@ -70,30 +70,22 @@
     public FileResponse convert(Param param) throws CommandValueException, ConvertException, ZipJfcException, IOException {
         FileResponse fileResponse = new FileResponse();
         PdfParam pdfParam = (PdfParam)param;
+
         StringBuilder stringCommand = new StringBuilder();
-        if (!pdfParam.isThumbnail() && !pdfParam.isMetadata()) {
-            stringCommand.append(generateImage(pdfParam));
-            runCommand(stringCommand.toString());
-        }
+        stringCommand.append(generateImage(pdfParam));
+        runCommand(stringCommand.toString());
+
         if (pdfParam.isThumbnail()) {
-            stringCommand = new StringBuilder();
-            stringCommand.append(generateImage(pdfParam));
-            runCommand(stringCommand.toString());
             stringCommand = new StringBuilder();
             stringCommand.append(generateThumbnail(pdfParam));
             runCommand(stringCommand.toString());
         }
-        if (pdfParam.isMetadata()) {
-            stringCommand = new StringBuilder();
-            stringCommand.append(generateImage(pdfParam));
-            runCommand(stringCommand.toString());
-            generateMetadata(pdfParam);
-        }
+
         if (param.isMetadata()) {
             MetadataConverter metadataConverter = new MetadataConverter();
```

Comment:
Is it correct the class name "MetadataConverter" ?, actually that class gets file's metadata

Reply:
yes, it is correct